### DISCUSSION TITLE
blog/after-the-offline-of-firewood

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#42] [Announcements]
# blog/after-the-offline-of-firewood

Philosophical Fragments and Academic Paths

https://pathos.page/blog/after-the-offline-of-firewood

**Comment by seeyousuperman** [comment]
很欣赏博主，思考能力、看穿本质能力、语言组织能力、以及文化水平，最重要的是，有智慧，本来躺下准备入睡，看了眼你的博客有更新，通读全文后，悬着的心才放下。

忍不住起来打开电脑进行评论，写了518个字的评论，本来还有更多想交流表达的，想了想，还是都删了。

**Comment by lg2it** [comment]
富有洞察力的思考，也引人深思。我想补充一点，一个人的思想与看待事物的角度与他的成长环境有很大的关系。倒不见得是愚蠢，更像是一种被某种话语体系裹挟而无法自拔。面对因为立场不同的极端表达，要么就尊重祝福，要么就道不同不相与谋吧。

**Comment by hujinyuan66** [comment]
我不知道积薪，我只是个普通人，只看对自己有用有意思的。

**Comment by webersongao** [comment]
他不算是“恨”吧，也没有报复社会，伤害他人之类的，只是批评而已。并且他挺喜欢四处走走拍照的，应该是爱“国”才是，其次，咱政府不允许民间结党，他连“党”也算不上。顶多算是个意见不同的个体户而已。
你这帽子扣得太大了，说人私自结党估计要判刑的。

**Comment by aetherrise** [comment]
非常客观有见地的一篇思考评论文，道不同，可以坐而论道，但以个人的不幸遭遇或者狭窄的认识就否定整个国家、整个民族，只能说是狂妄了。
作为纷争的当事方之一或者“导火索”，我对于李大毛这个人是有发言权的，毕竟大家都说最熟悉的你的人就是你的“敌人”。李大毛现实中确实遭遇到一些困难，看到过一些不公。某些时期，某级地方政府，可能确实存在问题，大家会愤怒不理智，这是人性，大家都可以接受并深表同情；但是冷静过后，偏激上升到攻击一切、否定一切的话，是非常可悲的，这样的人，这样的思维方式，在正常情况下大家已没法沟通了。
博主，你这篇文章我可以挂个链接到我网站文章内吗~

**Reply by Pathsis** [reply]
谢谢评论！没有问题。我也读了您的相关文章，获得了许多信息，很感谢！

**Comment by aetherrise** [comment]
收到，谢谢。很高兴能得到你的同意。-------- 原始邮件 --------发件人： 川崎 ***@***.***>日期： 2024年8月23日周五 16:38收件人： "Pathsis/pathos.page" ***@***.***>抄送： fatalor ***@***.***>, Comment ***@***.***>主    题： Re: [Pathsis/pathos.page] blog/after-the-offline-of-firewood (Discussion #42)
谢谢评论！没有问题。我也读了您的相关文章，获得了许多信息，很感谢！

**Comment by aetherrise** [comment]
我不想在别人的网站跟你吵，没有意义，你润你的日本，我过我的小日子。关于你说的一开始没有和你沟通的话，建议你去下载那份微信群聊记录，我有试图心平气和和你沟通，是你挑起的争端，我一开始就说了你移民日本，大家只会祝福，但是反过来攻击祖国就不行；关于考公和创业的事情，我没有捏造，都是你自己博客里的内容，那份群聊里也有截图；最后，关于基本逻辑的问题，我是想不到会有谁留言起名自己骂自己“只那人”的，还挂我的网站链接，有意思么。再再补充一点，我会在我网站发文也是因为你首先在你网站挂微信群的聊天记录，还就只有我和HowisHz的群聊名称不打码。微信群吵架是你发起的，后续的网站发文也是你起的头，听说前几天还把我信息挂到TG中立群了，所以，我举报你所有账号并不会感到任何亏欠。-------- 原始邮件 --------发件人： 李大毛 ***@***.***>日期： 2024年8月24日周六 15:59收件人： "Pathsis/pathos.page" ***@***.***>抄送： fatalor ***@***.***>, Comment ***@***.***>主    题： Re: [Pathsis/pathos.page] blog/after-the-offline-of-firewood (Discussion #42)
你要不要再补充两点事实，关于你从一开始就没打算和我“沟通”，然后举报了我所有你能找得到的账号这件事？
另外你是不是也忘了我在你网站留言指出了你关于我的几个事实错误，如“创业失败”等等，但你坚决不把评论放出来这件事？这就是你说的“最熟悉”我的人是吧。
你们一群基本逻辑能力都不具备的文盲就抱团取暖吧，我写了这么多相关主题的文章，没有一个人来就文章本身来批一下我的。全都是上升到国家、民族、群体，就是不谈细节。倒也不奇怪，真掰扯起来你们的逻辑全是不自洽的。

**Reply by aetherrise** [reply]
大家看群聊记录吧，对错每个人心中自有评论，你我在这争论没有意义。趁年轻有时间多搞搞技术，多赚点钱，别整天满身怨气怼天怼地，政治这门艺术普通人玩不转，这是真心话不管你爱不爱听。
还有最后的提醒，我光明正大实名备案，你敢把身份信息告诉帽子叔叔吗，你敢说以后不再踏入中国大陆半步吗，干不过就悄咪地躲起来好吧，蹦得越高摔得越惨。

**Comment by bosichong** [comment]
博主把我要想的但是写不出来的都系统的表达出来了，印象中看过大毛几篇博文，有种咽了一口苍蝇的感觉，但是又不好说什么，后来就不再看了。类似的博客我遇到过不是一两个了，果断的取关，正所谓志不同道不合。总是带着墨镜看世界，人也会变得不阳光了。国家也好党也好，我们每个人都会犯错，都会有瑕疵，但是我们还活着，至少没有去前线挖战壕运干粮，而且还能闲的蛋疼在这里打嘴炮，知足吧。

**Reply by jkjoy** [reply]
根本就不存在完美的政党与政治体系, 说到底就是看对人民的态度是如何的. 我也没觉得中国优秀,但是也不能说他不好.
毕竟生于斯,长于斯,也将老于斯.再多抱怨也是无益

**Comment by SungYinieh** [comment]
非常好文章，使我友链添加一位好作家。
